pkg/provider/cloud/nutanix/provider.go

```diff
@@ -150,9 +151,11 @@ func (n *Nutanix) reconcileCluster(cluster *kubermaticv1.Cluster, update provide
 }
 
 func deleteCategoryValues(client *ClientSet, cluster *kubermaticv1.Cluster) error {
+	ctx := context.TODO()
+
 	projectID, ok := cluster.Labels[kubermaticv1.ProjectIDLabelKey]
 	if ok {
-		_, err := client.Prism.V3.GetCategoryValue(ProjectCategoryName, projectID)
+		_, err := client.Prism.V3.GetCategoryValue(context.TODO(), ProjectCategoryName, projectID)
```

Comment:
```suggestion
		_, err := client.Prism.V3.GetCategoryValue(ctx, ProjectCategoryName, projectID)
```

Reply:
Fixed the highlighted problem, but as said, I'd much prefer to tackle TODO contexts across our provider implementations in a more coordinated fashion. I don't think this PR is the right place for that (I really just want to fix Nutanix cluster deletion right now).